scripts/functional.js

```diff
@@ -67,25 +61,9 @@ const buildComponentFixtures = async () => {
   return bundler.run();
 };
 
-const buildRuntimeFixtures = async () => {
```

Comment:
I decided to just use the alloyio.com functional test endpoint instead of building a new fixture. This allowed me to test running against multiple domains.

Reply:
Awesome decision.